last-call.xml

```diff
@@ -0,0 +1,22 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+  <channel>
+    <title>Ethereum EIPs Last Call Review</title>
+    <description>All EIPs which are in the two-week "last call" status, please help review these and provid your feedback!</description>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ site.url }}/last-call.xml" rel="self" type="application/rss+xml" />
+    {% assign eips = site.pages|where:"status","Last Call"|sort:"eip" %}
```

Comment:
This sort order may result in RSS/Atom readers missing any old EIPs that have just entered last call. I'm not sure if there's a way around it, though, unless Jekyll supports reading modification times of files.

Reply:
The correct solution is to sort based on Last Call review date, which *is* in the front matter.

Some solution might involve changing the way we put the dates into the front matter.

But I hope we can address that after #1100 is merged.